ui/app/(wrapped)/intentions/restitution/ConsoleSection/LineContent.tsx

```diff
@@ -3,28 +3,29 @@ import { Td } from "@chakra-ui/react";
 import { GraphWrapper } from "../../../../../components/GraphWrapper";
 import { TableBadge } from "../../../../../components/TableBadge";
 import { getTauxPressionStyle } from "../../../../../utils/getBgScale";
-import { getMotifLabel, MotifLabel } from "../../utils/motifDemandeUtils";
+import {
+  getMotifLabel,
+  MotifCampagne,
+  MotifLabel,
+} from "../../utils/motifDemandeUtils";
 import { getTypeDemandeLabel } from "../../utils/typeDemandeUtils";
 import { DemandesRestitutionIntentions } from "../types";
 
 export const LineContent = ({
   demande,
+  campagne,
 }: {
   demande: DemandesRestitutionIntentions["demandes"][0];
+  campagne?: string;
 }) => {
   const handleMotifLabel = (motif?: string[], autreMotif?: string) => {
```

Comment:
Mettre un s pour la nomenclature, et eviter les conflits avec plus bas

```suggestion
    (motifs?: string[], autreMotif?: string)
```

Reply:
Est ce que tu peux sortir la méthode du composant, comme ça, ça évite qu'elle soit recrééer à chaque render